docs/UserGuide.md

```diff
@@ -664,7 +664,7 @@ Only the remark “Hates carrots” is added to Annie Wan’s listing in the app
 
 ### View help: `help` <a name="help"></a>
 
-Shows a message explaining how to access the User Guide or obtain in-app guidance for selected commands.
+This command shows a message explaining how to access the User Guide or obtain in-app guidance for selected commands.
```

Comment:
Can consider changing to: "This command displays a message that shows you how you can access our User Guide or obtain in-app guidance for selected commands."